Examples/Framework/include/ActsExamples/Utilities/EventDataTransforms.hpp

```diff
@@ -15,7 +15,7 @@ namespace ActsExamples {
 
 ProtoTrack seedToPrototrack(const SimSeed &seed);
 
-std::optional<SimSpacePoint> findSpacePointForIndex(
+std::optional<const SimSpacePoint *> findSpacePointForIndex(
```

Comment:
hmmm how bad would it be to just return a raw pointer and null it if not found? optional of raw pointer seems redundant in some way

Reply:
actually you are right, I'll change it